v1/v1util/also_test.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2018 Google LLC All Rights Reserved.
```

Comment:
Also_test?

Reply:
I was going to call this `AlsoCloser` (after `NopCloser`) at first, but things got awkward when I needed both read and write.